tutorhastexo/templates/hastexo/build/guacd/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG HASTEXO_GUACD_VERSION={{ HASTEXO_GUACD_VERSION }}
+FROM guacamole/guacd:$HASTEXO_GUACD_VERSION
+
+{{ patch("guacd-dockerfile") }}
```

Comment:
I *think* if we're introducing a plugin-specific patch here, it should probably be prefixed by the plugin name (i.e. `hastexo-guacd-dockerfile`); would you agree?

Reply:
sure, sounds good.